### ISSUE TITLE
Adding a button on steven multi select

### ISSUE THREAD

**Issue by raja73** [isteven/angular-multi-select#552]
How can i add a button through which i can open and close the drop down?

**Comment by isteven** [comment]
By default it doesn't support such thing, but you can try to use "directive-id", and then find the element with class "checkboxLayer", add class "show" so it becomes "checkboxLayer  show".

**Comment by pankajparkar** [comment]
@rajaEmani  do you want a separate button that should open dropdown from outside isteven? Would you mind to elaborate the scenario, I'd love to help you on this. If you want you can also consider adding Stackoverflow question and add link here.

**Comment by rajaEmani** [comment]
@pankajparkar yes I want a separate button that should open dropdown from outside isteven. Please help me out on this

**Comment by pankajparkar** [comment]
Can you please share a plunker with what you have tried so far?

**Comment by rajaEmani** [comment]
I have seen your code.
Thanks,
Can you please show to how to add that event to an external button?

**Comment by rajaEmani** [comment]
@pankajparkar Any update on this or should I raise a stack over flow for you?

**Comment by pankajparkar** [comment]
@rajaEmani primarily I'd need plunker/fiddle with problem replication in it. So that I can try out stuff and would able make something work on it.
